browsers/safari.json

```diff
@@ -321,10 +321,17 @@
         "18": {
           "release_date": "2024-09-16",
           "release_notes": "https://developer.apple.com/documentation/safari-release-notes/safari-18-release-notes",
-          "status": "current",
+          "status": "retired",
           "engine": "WebKit",
           "engine_version": "619.1.26"
         },
+        "18.0.1": {
```

Comment:
This doesn't look right, we don't document patch versions.

@queengooborg Can you confirm?

Reply:
We don't, no, the browser data updater will need to account for patch versions it looks like.

Reply:
@queengooborg Here is the fix: https://github.com/mdn/browser-compat-data/pull/24652